tock/tock/templates/hours/timecard_form.html

```diff
@@ -11,7 +11,7 @@
     </p>
   </div>
 </div>
-{% if float_data %}
+{% if float_data is not None %}
```

Comment:
Why was this change needed?

Reply:
This was needed because there is a state where `float_data` is `None` and that's valid - in that case we don't want to display the Float information box at all.  What this change does is allows me to send an empty list instead and check for that - if there's no data, I can assume there was an error retrieving the information and provide a message.

Reply:
I think -- but I could be wrong -- that since Django templates are more truthy, an empty list still evaluates to none. So the prior way should still work. But it's probably not worth changing it back, either.

Reply:
Ah I'll have to check!  That said, this is also more explicit. :-)